lib/db/mem.js

```diff
@@ -38,7 +38,7 @@ var DEVICE_FIELDS = [
   'callbackIsExpired'
 ]
 
-var SESSION_FIELDS = [
+const SESSION_DEVICE_FIELDS = [
```

Comment:
This is unrelated change that confused me while working on this PR. These are session device fields that are returned when updating/retrieving devices.